content/pages/va-gov-redirect.html

```diff
@@ -0,0 +1,10 @@
+---
+production: false
+---
+<!doctype html>
+<html>
+  <head>
+      <meta http-equiv=refresh content="1; url=https://staging.va.gov">
+      <link rel=canonical href="/">
+  </head>
+</html>
```

Comment:
This will be used for validating that the onboarding modal is working, while the full Vets.gov-to-VA.gov redirects are being implemented.